sys/sys/types.h

```diff
@@ -300,6 +300,16 @@ typedef	__uint64_t	uoff_t;
 typedef	char		vm_memattr_t;	/* memory attribute codes */
 typedef	struct vm_page	*vm_page_t;
 
+#ifdef COMPAT_FREEBSD32
+typedef	int32_t		intptr32_t;
```

Comment:
Hmm, other definitions use `__int32_t`, etc. using the non-underscore version seems fine though since it's defined above.

Reply:
I'll switch to double underscore.